perf/config.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2020 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Configuration for perf event manager.
+package perf
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"time"
+
+	"k8s.io/klog"
+)
+
+type Events struct {
+	// List of perf events to be measured. Events in this list
+	// will not be grouped. See group_fd argument documentation
+	// at man perf_event_open.
+	NonGrouped []Event `json:"non_grouped"`
+
+	// List of groups of events to be measured. See group_fd
+	// argument documentation at man perf_event_open.
+	Grouped [][]Event `json:"grouped"`
+
+	// Interval between each measurement.
+	Interval Duration `json:"interval"`
```

Comment:
Unless we have a good reason to, I would prefer using the housekeeping interval with jitter like we do for other metrics.  Otherwise, starting many containers at once (a common pattern) leads to large spikes in usage from cAdvisor collecting all container metrics at once.

Reply:
:+1: